### DISCUSSION TITLE
What is the version at the .org site?

### DISCUSSION THREAD

**Discussion by lduperval** [Tichau/FileConverter#394] [General]
I found another version of this on a filedecoder.org which supports AVIF (which is one I need). Does anyone know what that version is and how it's realted to this? Is it a fork?

Thanks,

L

**Comment by JackySw** [comment]
I did not install it, but you mean https://file-converter.org/ 
But indeed, is it a scam or legit? It has a triple amount of converting formats (the page say over a 1000), even docs, ebooks and CAD. But it also refers to ZamZar in the text which is a online converting tool? Which is nice, but do you trust uploading your files and then getting them back without knowing what happened to them? I think that the .org domain might have been expired and picked up and is now used in a 'smart way' by the ZamZar people.

**Reply by thelogicmatrix** [reply]
Yeah I suspect the same, I  check the bottom of the site and saw this at the bottom.
![image](https://github.com/Tichau/FileConverter/assets/20381195/765b65ea-eeb0-48f2-a63f-3f23013ee5b7)
The icon is ever so slightly different. It's honestly quite predatory but is ok for the general user, but I'm happy I found this way before Zamzar stole the site because the granularity and ability to adjust stuff is second to none on Tichau's file converter

**Reply by JustinDake** [reply]
Found this page after seeing Reddit suggest the .org and immediately getting scammy vibes from the website. I notice that although the homepage eludes to a paid version, there is no paid portal anywhere to be found. So where do you expect a person to pay? Even searching site:file-converter.org only turns up 2 pages, the homepage and a Downloads page which says the software has been in development since 2006, which is when Zamzar's Wiki says it was founded. If the .org site truly has nothing to do with Adrien Allard, that's predatory at best and a total scam at worst. On the other hand, if the converter we love has actually been sold to Zamzar, I'd think we deserve to know that.

**Reply by thelogicmatrix** [reply]
Nah I think it's probably a case of the website getting yanked and taken by zamzar or more likely the day to day keepin g of the website got expensive and the opensource version guy stopped hosting the folks at zamzar took it as an opportunity to scam some people

**Comment by bdan-codes** [comment]
so this still exists... (just under a different url)
https://file-converter.org

**Reply by bdan-codes** [reply]
also on this scam website it costs money

**Reply by bdan-codes** [reply]
on top of that it infringes on the rights/copyright rules listed on this github page. it obviously modifed the icons of the right click context menu, based on the screenshots shown on the scam page. so...

**Comment by NastyFlytrap** [comment]
I fell for this too. I watched the chris titus video, googled it, and i didnt even realize until i made a comment on the video and someone told me that i should use the link he put in the comments instead, and now im here

Zamzar are fradulent fucks and they need to eat shit for this.
